apps/fluent-tester/src/FluentTester/TestComponents/ButtonExperimental/ButtonIconTestSection.tsx

```diff
@@ -67,6 +61,14 @@ export const ButtonIconTest: React.FunctionComponent = () => {
       <Button icon={testImage} style={commonTestStyles.vmargin}>
         PNG
       </Button>
+      {svgIconsEnabled && (
+        <Button style={commonTestStyles.vmargin} icon={{ svgSource: svgProps }}>
+          Icon Button and Chevron
+          <View style={{ paddingStart: 4 }}>
```

Comment:
Why the new inline style?

Reply:
This is to add spacing before the chevron icon. Since it's a child passed into the Button, it needs to be explicitly styled. The Button won't do anything with it and just render it as is.

Would you like me to move the style into a stylesheet?

Reply:
Normally yes, because of this comment on one of my first reviews in FURN: https://github.com/microsoft/fluentui-react-native/pull/372#discussion_r474326015

... buuuut we create subobjects in a lot of places (Like line 36), so it's not new.

Reply:
Not sure if that link worked so screenshot it is. To be clear, I don't consider this review blocking.
![image](https://user-images.githubusercontent.com/6722175/152248459-1f34813b-4596-4f86-9682-57e9a74a61ee.png)

Reply:
Yeah, Jason's right that this isn't great for render performance, but you're right that we also do this all over the code already.

I'll put it in a stylesheet/const object for cleaner code sake.